package.json

```diff
@@ -18,7 +18,7 @@
     "@shopify/app-bridge": "^3.1.0",
     "@shopify/app-bridge-react": "^3.1.0",
     "@shopify/app-bridge-utils": "^3.1.0",
-    "@shopify/polaris": "^10.42.0",
+    "@shopify/polaris": "^10.42.1",
```

Comment:
Can we bump this to `^10.45.0`? We recently shipped an color adjustment to the `Tabs` component.